### ISSUE TITLE
rocky 9 - authconfig replaced by authselect in kickstart config

### ISSUE THREAD

**Issue by bashtian-fr** [lavabit/robox#281]
While using rocky9 vagrant kickstart file I noticed that it uses `authconfig` which has been replaced by `authselect`. Hence Anaconda will fail to install that `authconfig` package.

The solution is to replace both occurrences of `authconfig` by `authselect` (But beaware that while `authselect` supports the 2 params and can be a drop-in replacement, it works differently)

**Comment by ladar** [comment]
@bashtian-fr I would this change applies to all of the RHEL 9 variations? And probably to the Fedora boxes. Just a question of starting with which release?

Can you summarize how the behavior changes?
